test/test_json_util.py

```diff
@@ -270,6 +271,52 @@ def test_regex(self):
             json_util.dumps(Regex('.*', re.M | re.X),
                             json_options=LEGACY_JSON_OPTIONS))
 
+    def test_regex_pickling(self):
```

Comment:
These tests don't belong in test_json_util since it's not related to JSON. I'd suggest moving them to test_bson or to the individual files like test_timestamp/test_dbref.

We also need tests for Int64, MinKey, and MaxKey.

Reply:
Done.